packages/hooks/src/NFT/useNFTViewMethod.ts

```diff
@@ -0,0 +1,70 @@
+import useSWR from 'swr'
+import { networkId, nodeUrl } from '../near-utils'
+import { connect, Contract } from 'near-api-js'
+
+export type useNFTViewMethodType = {
+    error?: string
+    loading: boolean
+    data?: any
+}
+
+const fetchNFTView = async (
+    contractAddress: string,
+    methodName: string,
+    serializedParams: string
+) => {
+    const near = await connect({
+        networkId,
+        nodeUrl,
+        deps: {
+            //@ts-ignore
+            keyStore: undefined,
+        },
+    })
+    //@ts-ignore
+    const account = await near.account(null)
+
+    const contract = await new Contract(account, contractAddress, {
+        viewMethods: [
```

Comment:
There should be a file to get the contract methods from (`near-utils.ts`). You just have to pass it a parameter that says `nft`. Just to remove some boilerplate code here.